api/data_refinery_api/management/commands/post_downloads_summary.py

```diff
@@ -122,7 +121,7 @@ def should_display_email(email: str) -> bool:
 
 def get_ip_location(remote_ip):
     try:
-        city = requests.get("https://ipapi.co/" + remote_ip + "/json/", timeout=10).json()["city"]
+        data = requests.get("https://ipapi.co/" + remote_ip + "/json/", timeout=10).json()
```

Comment:
It just occurred to me that we're doing this at summary time only. This means we don't have the address stored anywhere. IP addresses change over time, which means that the only persistent store of locations we've had downloads from is the physcial maps in our office?

Reply:
hmm does the location of an IP address change?

Reply:
Yes, apparently by about 10% by year: https://stackoverflow.com/questions/23493812/are-geolocation-ip-ranges-changing-over-the-years

This means that if we ever wanna look back at our downloads over the years our location data won't be useful.

Reply:
ah, good find! In that case, let me create a different issue to save these locations when we calculate them.

Reply:
created https://github.com/AlexsLemonade/refinebio/issues/2217